data/experiment-recipe-samples/mobile-a-a.json

```diff
@@ -0,0 +1,29 @@
+{
+  "slug": "mobile-a-a-example",
+  "application": "reference-browser",
+  "userFacingName": "Mobile A/A Example",
+  "userFacingDescription": "An A/A Test to validate the Rust SDK",
+  "isEnrollmentPaused": false,
+  "bucketConfig": {
+    "randomizationUnit": "nimbus_id",
+    "namespace": "mobile-a-a-example",
+    "start": 0,
+    "count": 5000,
+    "total": 10000
+  },
+  "startDate": null,
+  "endDate": null,
+  "proposedEnrollment": 7,
```

Comment:
What is the client SDK expected to do with this "proposed enrollment" field, if anything?